js/src/lib/inferenceSnippets/serveCurl.ts

```diff
@@ -1,41 +1,51 @@
 import type { PipelineType, ModelData } from "../interfaces/Types";
 import { getModelInputSnippet } from "./inputs";
 
-export const bodyBasic = (model: ModelData): string =>
-	`-d '{"inputs": ${getModelInputSnippet(model, true)}}'`;
+export const snippetBasic = (model: ModelData, accessToken: string): string =>
+	`curl https://api-inference.huggingface.co/models/${model.id} \\
+	-X POST \\
+	-d '{"inputs": ${getModelInputSnippet(model, true)}}' \\
+	-H "Authorization: Bearer ${accessToken || `{API_TOKEN}`}"
+`;
 
-export const bodyZeroShotClassification = (model: ModelData): string =>
-	`-d '{"inputs": ${getModelInputSnippet(model, true)}, "parameters": {"candidate_labels": ["refund", "legal", "faq"]}}'`;
+export const snippetZeroShotClassification = (model: ModelData, accessToken: string): string =>
+	`curl https://api-inference.huggingface.co/models/${model.id} \\
+	-X POST \\
+	-d '{"inputs": ${getModelInputSnippet(model, true)}, "parameters": {"candidate_labels": ["refund", "legal", "faq"]}}' \\
+	-H "Authorization: Bearer ${accessToken || `{API_TOKEN}`}"
+`;
+
+export const snippetFile = (model: ModelData, accessToken: string): string =>
+	`curl https://api-inference.huggingface.co/models/${model.id} \\
+	-X POST \\
+	--data-binary '@${getModelInputSnippet(model, true, true)}' \\
+	-H "Authorization: Bearer ${accessToken || `{API_TOKEN}`}"
+`;
 
 export const curlSnippetBodies:
```

Comment:
If we rename to `snippetBasic` etc., let's maybe rename this to `curlSnippets` ?

Reply:
great suggestion! handled in https://github.com/huggingface/hub-docs/pull/106/commits/79a96b7601c4f5119fcd5225e02838e23ac68fa2